src/Mod/Path/Gui/Resources/Path.qrc

```diff
@@ -111,6 +112,7 @@
         <file>panels/PageOpSlotEdit.ui</file>
         <file>panels/PageOpSurfaceEdit.ui</file>
         <file>panels/PageOpWaterlineEdit.ui</file>
+	<file>panels/PageOpVcarveEdit.ui</file>
```

Comment:
There's a tab here, it should be spaces.

Reply:
fixed